python/ray/_private/runtime_env/packaging.py

```diff
@@ -144,18 +145,32 @@ def handler(path: Path):
     return hash_val
 
 
-def parse_uri(pkg_uri: str) -> Tuple[Protocol, str]:
+class UriType(Enum):
+    REMOTE = (auto(),)
+    USER_LOCAL = (auto(),)
+    CLUSTER_LOCAL = (auto(),)
```

Comment:
Can we add a comment documenting the meaning of these?  Or we can just say "see the parse_uri docstring" if that's easier

Reply:
For my knowledge what's the difference between `(auto(),)` and just `auto()`?